src/components/Column/Column.tsx

```diff
@@ -40,40 +46,47 @@ export class Column extends React.Component<ColumnProps, ColumnState> {
       cards: [],
       lastIndex: 0,
       name: this.props.name,
-      isEditing: false,
+      isEditing: !!this.props.isEditing ? true : false,
     };
 
     this.nameInput = React.createRef();
   }
 
-  componentWillMount() {
+  componentDidMount() {
+    const sessionId = sessionStorage.getItem("retroSessionId") || "";
```

Comment:
This is more of a general React question - when not using Redux or a state management library, is the preference to pass stuff like this down from as high as possible (e.g. load the `sessionId` in the `Board` component and pass as a prop into its children) or to "sideload" like this to keep the data close to the usage?

Reply:
I've chosen to simply access this sessionStorage value wherever I need it rather than passing it around through some other mechanism.